desserts/mango-coconut-rice.md

```diff
@@ -0,0 +1,31 @@
+# Mango coconut rice
+
+This Thai dessert compliments salty and spicy food well. This recipe is inspired by a Food Network [mango coconut rice recipe](https://www.foodnetwork.com/recipes/mango-coconut-rice-recipe-1937791) from Paula Dean and a [caramelized mango recipe](https://www.myrecipes.com/recipe/caramelized-mangoes) from MyRecipes.
+
+
+## Ingredients
+
+- 1 tablespoon olive oil
+- 1.5 cups long-grain rice (ideally sushi rice)
+- 1 can (14 ounces) unsweetened coconut milk
+- 1/2 cup + 1.5 tablespoons water
+- 2 tablespoons sugar
+- 1 teaspoon salt
+- 1 large, ripe mango, peeled and cubed (about 2 cups)
+
+
+## Instructions
+
+1. Peel the ripe mango and cut into cubes, about 1/3 to 1/2 inch per side.
+
+2. Combine 2 tablespoons sugar and 1.5 tablespoons water in a medium pan over medium-high heat; cook until sugar dissolves. Continue cooking 3 minutes or until golden; do not stir. Reduce heat to medium.
+
+3. Add cubed mango to pan, tossing gently. Cook 10 minutes or until mango is lightly browned, stirring frequently.
```

Comment:
So my idea of caramelizing mangoes would only apply to frozen and unripe mango. When they're ripe they are already sweet and probably don't need any more sugar.